serverless.yml

```diff
@@ -0,0 +1,24 @@
+service: bash
+
+provider:
+  name: aws
+  stage: production
+
+layers:
+  bash:
+    name: bash
+    description: Bash custom runtime
+    path: .
+
+package:
+  include:
+    - index.sh
+
+functions:
+  check-bash-layer:
```

Comment:
STEP 2: Serverless parses config file.

It determines the runtime is provided meaning a lambda layer provides it. In this case it's bash. The bootstrap file has the logic. In that logic it will determine that handler: index.test means this lambda function will execute test bash function in index.sh.